tests/test_test.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from _pytest.capture import CaptureFixture
+from _pytest.monkeypatch import MonkeyPatch
+
+import rasa.model
+import rasa.cli.utils
+
+
+def monkeypatch_get_latest_model(tmp_path: Path, monkeypatch: MonkeyPatch):
+
+    latest_model = tmp_path / "my_test_model.tar.gz"
+
+    def mock_get_latest_model():
+        return str(latest_model)
+
+    monkeypatch.setattr(rasa.model, "get_latest_model", mock_get_latest_model)
+
+
+def test_test_core_models_in_directory_input_default(
+    capsys: CaptureFixture, tmp_path: Path, monkeypatch: MonkeyPatch
+):
+    from rasa.test import _test_core_models_in_directory_input
+
+    monkeypatch_get_latest_model(tmp_path, monkeypatch)
+
+    latest_model = Path(rasa.model.get_latest_model())
```

Comment:
let's add a short comment so that other developers reading the test know why this is done
```suggestion
    # Create a fake model on disk so that `is_file` returns `True`
    latest_model = Path(rasa.model.get_latest_model())
```